backend/parser.py

```diff
@@ -49,7 +49,7 @@ def add_point(self, timestamp, data: list, latency=None):
         else:
```

Comment:
For the data attribute, only declare the channel if there is data to add. If a sensor reports one data point, there should not be 2 extra empty ones.

Reply:
Also, the comments of "x/y/z data are not accurate for all sensors.

Reply:
Should I have an __init__ arg that allows the user to specify how many channels or would it be best to just parse that from the file automatically

Reply:
It should be smart enough to parse automatically.